ballerina/modules/soap12/tests/http_soap_service.bal

```diff
@@ -49,16 +63,44 @@ service / on new http:Listener(9090) {
                 signatureAlgorithm: soap:RSA_SHA256,
                 decryptionAlgorithm: soap:RSA_ECB,
                 decryptionKey: serverPrivateKey
-            }, payload);
+        }, payload);
         xml securedEnv = check soap:applySecurityPolicies(
             {
                 signatureAlgorithm: soap:RSA_SHA256,
                 encryptionAlgorithm: soap:RSA_ECB,
                 signatureKey: serverPrivateKey,
                 encryptionKey: clientPublicKey
-            }, applyOutboundConfig);
+        }, applyOutboundConfig);
         http:Response response = new;
         response.setPayload(securedEnv);
         return response;
     }
+
+    resource function post getSecuredMimePayload(http:Request request) returns http:Response|error {
+        xml payload = check (check request.getBodyParts())[0].getXml();
+        xml applyOutboundConfig = check soap:applyOutboundConfig(
+            {
+            verificationKey: clientPublicKey,
+            signatureAlgorithm: soap:RSA_SHA256,
+            decryptionAlgorithm: soap:RSA_ECB,
+            decryptionKey: serverPrivateKey
+        }, payload);
+        xml securedEnv = check soap:applySecurityPolicies(
+            {
+            signatureAlgorithm: soap:RSA_SHA256,
+            encryptionAlgorithm: soap:RSA_ECB,
+            signatureKey: serverPrivateKey,
+            encryptionKey: clientPublicKey
+        }, applyOutboundConfig);

```

Comment:
Check formatting in these places. (Might be a bug in the Ballerina formatter.

Reply:
Fixed in [14d1d13](https://github.com/ballerina-platform/module-ballerina-soap/pull/112/commits/14d1d137c0db3e9ae9ef1bbf1fbf860c4635fd00)